apps/site-landing/src/containers/App/index.tsx

```diff
@@ -34,11 +34,7 @@ export const App = () => {
         <MainNavigation />
 
         <Container className={styles.appContainer}>
-          <MainPageTransition>
-            <section id={styles.mainContent} className={styles.pageMainInner}>
-              <Routes />
-            </section>
-          </MainPageTransition>
+          <MainPageTransition routes={routes} />
```

Comment:
Will rename.  The transition class needed to manage the Switch to transition correctly.  Moved switch inside the Transition class to ensure it's fixed everywhere.  Will rename to "MainContentRouter" or something similar to indicate it's increased responsibilities